src/lib/viewers/doc/_docBase.scss

```diff
@@ -295,6 +295,17 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         }
     }
 
+    //overwriting background image in pdf_viewer.css. Was causing 404's
+    /* stylelint-disable declaration-no-important */
+    .pdfViewer .page.loadingIcon {
+        background-image: none !important;
```

Comment:
Is `!important` the only way to do this? I know in general we try to avoid using this

Reply:
yup reworked this